### ISSUE TITLE
Strings from TrustedType columns getPropertyType/getAttributeType tables should be quoted

### ISSUE THREAD

**Issue by fred-wang** [w3c/trusted-types#552]
These functions return a DOMString taken from a specific column of a table, containing links to TrustedHTML/TrustedScript/TrustedScriptURL interfaces (which to add more confusion, have a stringifier returning the associated string data). IIUC the spec means to return a string with the interface names instead. So these interface names should probably be quoted, to be consistent with other strings in these tables.

cc @lukewarlow

**Comment by lukewarlow** [comment]
That would break 'Get compliant attribute value' which uses the table too but does want an interface. So in this case I think it's better to just return something like 'expectedType's interface name'

**Comment by fred-wang** [comment]
Good point, yeah reformulating expectedType's interface name for getAttributeType, and the same for getPropertyType sounds good.

**Comment by fred-wang** [comment]
@lukewarlow That said, I don't see where https://w3c.github.io/trusted-types/dist/spec/#validate-attribute-mutation is actually used ¿?

**Comment by lukewarlow** [comment]
In a pull request to the DOM spec.
